net/http/inc/THttpServer.h

```diff
@@ -44,7 +44,8 @@ class THttpServer : public TNamed {
    TString fDefaultPageCont; ///<! content of the file content
    TString fDrawPage;        ///<! file name for drawing of single element
    TString fDrawPageCont;    ///<! content of draw page
-   TString fCors;            ///<! CORS (cross-origin resource sharing): sets Access-Control-Allow-Origin for ProcessRequest responses
+   TString
+      fCors; ///<! CORS (cross-origin resource sharing): sets Access-Control-Allow-Origin for ProcessRequest responses
 
```

Comment:
This one looks weird to me...

Reply:
OK, just ignore my comment...

Reply:
If I will put it in one line, Travis CI will complain